src/app/shared/services/login/login.service.ts

```diff
@@ -45,11 +45,16 @@ export class LoginService {
   }
 
   isSessionActive() {
-    return (Boolean(localStorage.getItem(PROVIDER_TYPE)) && Boolean(localStorage.getItem(PUBLIC_KEY)));
+    return (
+      Boolean(localStorage.getItem(PROVIDER_TYPE)) 
+      && Boolean(localStorage.getItem(PUBLIC_KEY)) 
+      && Boolean(localStorage.getItem(IS_ETH_SIGNER))
```

Comment:
is `IS_ETH_SIGNER` const used somewhere inside iam-client-lib? The same with `PUBLIC_KEY` ? It's more like a question, if we need to handle it inside library as this is for storing session.

Reply:
Yes, iam-lib uses both these. `IS_ETH_SIGNER` is used to correctly sign message and `PUBLIC_KEY` is needed by did-reg, but might be deprecated later

Reply:
@JGiter I think we can remove `PUBLIC_KEY` as we're no longer requiring that it be added to a DID document, right? We can do this in a separate story though

Reply:
@jrhender  That is right, but first we need to change `EwSigner` to not require public key

Reply:
Ah okay, makes sense, thanks @JGiter